app/controllers/ems_common.rb

```diff
@@ -427,27 +427,66 @@ def update_button_reset
 
   def update_button_validate
     verify_ems = find_by_id_filtered(model, params[:id])
-    validate_credentials verify_ems
+    queue_credential_validation_task verify_ems
   end
   private :update_button_validate
 
-  def validate_credentials(verify_ems)
+  def queue_credential_validation_task(verify_ems)
     set_record_vars(verify_ems, :validate)
+    creds = set_record_creds(verify_ems)
+
     @in_a_form = true
     @changed = session[:changed]
 
     # validate button should say "revalidate" if the form is unchanged
     revalidating = !edit_changed?
-    result, details = verify_ems.authentication_check(params[:type], :save => revalidating)
```

Comment:
The `:save => revalidating` needs to be included in the args as well.  Then, [the call to `ems.update_authentication`](https://github.com/ManageIQ/manageiq/pull/4869/files#diff-00124749a6f7c1abd2a1adaae2a77e74R49) can use that argument to determine if it needs to save the authentication credentials on validation or not.